nservicebus/hosting/windows-service.md

```diff
@@ -107,6 +107,17 @@ sc.exe config [ServiceName] depend= <Dependencies(separated by / (forward slash)
 sc.exe config SalesEndpoint depend= MSMQ/MSDTC/RavenDB
 ```
 
+### Restart Recovery
+
+The endpoint instance may decide to quit due to its internal circuit breaker behavior. When an NServiceBus endpoint is hosted as a Windows Service is it important to configure the Windows Service Recovery options. These can be set by the service properties or via `sc.exe` which has advanced configuration options.
```

Comment:
When using the host it is the default behavior. When self hosting the instance becomes a zombie. I'll try and improve this.